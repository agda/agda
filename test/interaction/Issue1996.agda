{-# OPTIONS --no-keep-pattern-variables #-}

open import Agda.Builtin.Nat
open import Agda.Builtin.Equality

test : (x : Nat) (f : {n : Nat} → Nat) → f {0} ≡ x → Nat
test x f p = {!p!}
