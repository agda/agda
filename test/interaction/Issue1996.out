(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Nat" nil)
((last . 1) . (agda2-goals-action '(0)))
((last . 2) . (agda2-make-case-action '("test .f f refl = ?")))
((last . 1) . (agda2-goals-action '(0)))
